### ISSUE TITLE
Edge case alerts when changing mass need some corrections

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#90]
As I mentioned in https://github.com/phetsims/gravity-force-lab/issues/135#issuecomment-456363119, these edge case alerts have been correctly implemented in REGULAR, but are not always correct over here in BASICS. This might be due to lack of time for @mbarlow12 to finish these edge cases for both sims.

Here are screen shots of BASICS showing where the alerts are correct and incorrect.

@zepumph, please make the alerts work correctly like in REGULAR. Once they are correct, we can see where we will need tweaks to better align visual movement and the description of the visual movement.

**Incorrect: Mass 1's mass increases and pushes Mass 2 right**
Alert incorrectly describes mass 1 as moving left.
<img width="614" alt="mass1-increase-moves-mass2right-incorrect-alert" src="https://user-images.githubusercontent.com/1902116/51530084-7a483600-1e3a-11e9-8e19-9f79958ce76f.png">
<img width="614" alt="mass1-increase-pushes-mass2-right-incorrect alert" src="https://user-images.githubusercontent.com/1902116/51530088-7caa9000-1e3a-11e9-97e2-fd663a618fee.png">

**Incorrect: Mass 2 pushes Mass 1 left**
Alert incorrectly describes mass 2 as moving right.
<img width="617" alt="01-mass2-increases-pushes-mass1-incorrect-alert" src="https://user-images.githubusercontent.com/1902116/51530639-baf47f00-1e3b-11e9-8919-3af01cc13116.png">
<img width="617" alt="02-mass2-increases-pushes-mass1-incorrect-alert" src="https://user-images.githubusercontent.com/1902116/51530649-bf209c80-1e3b-11e9-9b0a-8db4a45c6d0d.png">

**Incorrect alert when changing sphere is at edge pushing other mass away**
<img width="618" alt="incorrect-when-mass1-changes-at-edge" src="https://user-images.githubusercontent.com/1902116/51531535-014add80-1e3e-11e9-806d-eaab6fb09502.png">

**Incorrect alert: Same issue when increasing mass of Mass 2 when Mass 2 is at right edge**
<img width="616" alt="incorrect-mass2-at-edge-mass2-increase" src="https://user-images.githubusercontent.com/1902116/51531652-5a1a7600-1e3e-11e9-9af2-6d93b5fc18d7.png">

**Correct alert when unchanging sphere is at edge**
**Increasing mass of Mass 1 when Mass 2 is at right edge.** In this case the alert is correctly describes mass 1 as moving left.
<img width="618" alt="correct-mass1-moves-left-at-edge" src="https://user-images.githubusercontent.com/1902116/51530410-268a1c80-1e3b-11e9-9f16-676a9953c1f0.png">

**Increasing mass of Mass 2 when Mass 1 is at left edge.** Again, alert correctly describes mass 2 as moving right.
<img width="619" alt="mass2-correct-alert-mass2-increase-mass1-at-left-edge" src="https://user-images.githubusercontent.com/1902116/51531202-23902b80-1e3d-11e9-9172-c5194f24d1c9.png">
<img width="618" alt="screen shot 2019-01-22 at 11 53 08 am" src="https://user-images.githubusercontent.com/1902116/51531213-268b1c00-1e3d-11e9-9a71-e9a695a3b7d6.png">

**Comment by terracoda** [comment]
@zepumph, if you can get the description going in the correctly, great if not - do not worry. 
Things are pretty good, at least movement is being described, just not quite the right direction yet.

**Comment by zepumph** [comment]
This was a pretty big change. It seems to be working as expected now. Please review.

**Comment by zepumph** [comment]
Sometimes it seems like there is a key press delay, where the "pushed mass" value text is one key press behind. But I can't get that to occur consistently, and it doesn't seem too distracting for right now. Let me know what you think.

**Comment by terracoda** [comment]
@zepumph, I verified that all the alerts are firing correctly.
Shall we close this issue and start a new one to optimize the visual movement and better synchronizing the alerts about which mass is moving?

**Comment by terracoda** [comment]
Actually, I will go ahead and do just that - start a new issue.
